server-rbpi3/update.sh

```diff
@@ -102,7 +102,7 @@ if [ -e /var/www/html_old_"$NOW"/src/SuplaBundle/Command/SimulateEventsCommand.p
 fi
 
 # Cleaning unnecessary post-installation files
-rm -fr var/cache/prod
+rm -fr var/cache/*
```

Comment:
If you call it as root then you will have a problem with permissions.
Therefore, after php bin / console cache: warmup
you should also add chown -R www-data: www-data ./var/cache